smoke/README.md

```diff
@@ -0,0 +1,105 @@
+<pre>
+Feature: Fight or flight
+  In order to increase the ninja survival rate,
+  As a ninja commander
+  I want my ninjas to decide whether to take on an
+  opponent based on their skill levels
+
+  Scenario: Weaker opponent
+    Given the ninja has a third level black-belt
+     When attacked by a samurai
+     Then the ninja should engage the opponent
+
+  Scenario: Stronger opponent
+    Given the ninja has a third level black-belt
+     When attacked by Chuck Norris
+     Then the ninja should run for his life
+</pre>
+Python Step Implementations
+Steps used in the scenarios are implemented in Python files in the “steps” directory. You can call these whatever you like as long as they use the python *.py file extension. You don’t need to tell behave which ones to use - it’ll use all of them.
+
+The full detail of the Python side of behave is in the [API documentation](https://behave.readthedocs.io/en/stable/api.html).
+
+Steps are identified using decorators which match the predicate from the feature file: given, when, then and step (variants with Title case are also available if that’s your preference.) The decorator accepts a string containing the rest of the phrase used in the scenario step it belongs to.
+
+Given a Scenario:
+<pre>
+Scenario: Search for an account
+   Given I search for a valid account
+    Then I will see the account details
+</pre>
+Step code implementing the two steps here might look like (using selenium webdriver and some other helpers):
+<pre>
+@given('I search for a valid account')
+def step_impl(context):
+    context.browser.get('http://localhost:8000/index')
+    form = get_element(context.browser, tag='form')
+    get_element(form, name="msisdn").send_keys('61415551234')
+    form.submit()
+
+@then('I will see the account details')
+def step_impl(context):
+    elements = find_elements(context.browser, id='no-account')
+    eq_(elements, [], 'account not found')
+    h = get_element(context.browser, id='account-head')
+    ok_(h.text.startswith("Account 61415551234"),
+        'Heading %r has wrong text' % h.text)
+</pre>
+Openshift dedicated python modules [openshift.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/openshift.py), [command.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/command.py), [project.py](https://github.com/openshift/csi-driver-shared-resource/tree/master/smoke/features/steps/project.py).These modules are used as a wrapper for openshift cli,one can create the Openshift object & use the functionalities to check the resources.
+<pre>
+└── steps
+    ├── command.py
+    ├── openshift.py
+    ├── project.py
+</pre>
+Visit [Behave](https://behave.readthedocs.io/en/stable/tutorial.html) for more info.
+# Test Results 
+<pre>- The test results are JUnit files generated for each feature & are collected in out dir post test run is complete
```

Comment:
Do you really want the sentence here bracketed by `<pre> </pre>`

It looks weird when a view this file from your PR in a browser, with markdown rendered.

Probably should remove the pre's

Reply:
Hard agree here - use triple-backticks instead.